base_layer/key_manager/src/lib.rs

```diff
@@ -30,3 +31,7 @@ pub(crate) fn mac_domain_hasher<D: Digest + LengthExtensionAttackResistant>(
 ) -> DomainSeparatedHasher<D, KeyManagerDomain> {
     DomainSeparatedHasher::<D, KeyManagerDomain>::new_with_label(label)
 }
+
+pub fn get_birthday_from_unix_epoch(birthday: u16, to_days: u16) -> u64 {
+    u64::from(birthday.saturating_sub(to_days)) * 24 * 60 * 60 + BIRTHDAY_GENESIS_FROM_UNIX_EPOCH
+}
```

Comment:
Recommend a function-level comment for this, to indicate its purpose. Might also be helpful to rename it in a way that indicates an extra offset is present and clearly describes the unit conversion.

Reply:
I added some documentation and add `_in_seconds` to the naming of the function